concepticondata/conceptrelations.tsv

```diff
@@ -843,4 +843,16 @@ SOURCE	SOURCE_GLOSS	RELATION	TARGET	TARGET_GLOSS
 4056	CHAOS	antonymof	1998	COMMAND
 1083	MESS	instanceof	4056	CHAOS
 4056	CHAOS	resultsin	3853	CONFUSION
-4056	CHAOS	resultsin	781	FEAR (FRIGHT)
\ No newline at end of file
+4056	CHAOS	resultsin	781	FEAR (FRIGHT)
+4058	ACCIDENT	similar	1239	MISTAKE
+4060	PASSION	antonymof	4052	INDIFFERENT
+4060	PASSION	instanceof	923	LOVE
+3761	EMOTIONAL	instanceof	4060	PASSION
+4060	PASSION	instanceof	1880	PASSIONATE
+2542	HERB	instanceof	4061	PARSLEY
+4062	FUN	similar	1413	PLAY
+4062	FUN	antonymof	905	BORING
```

Comment:
The propoer antonym would be BOREDOM, and I think we don't have this concept yet.